gqlalchemy/memgraph.py

```diff
@@ -585,3 +585,50 @@ def get_lambda(expression: str) -> str:
             return ""
         else:
             return f"(e, v | {expression})"
+
+
+class BreadthFirstSearch(IntegratedAlgorithm):
+    """Build a BFS call for a Cypher query.
+
+    The Breadth-First Search can be called in Memgraph with Cypher queries
```

Comment:
The **Breadth-first search** by Wikipedia. When we use shorter-term, then it's all uppercase (BFS).